### DISCUSSION TITLE
new gallery keybinds and enlarging image?

### DISCUSSION THREAD

**Discussion by coldasicee** [lllyasviel/stable-diffusion-webui-forge#1086] [Q&A]
the new gallery makes the user change the image to the next or previous one if you click more to the right/left of it's preview so I can't enlarge the image like before. What changed and how can I see the "fullscreen" image like before? (click on the image would put it in fullscreen)

**Comment by zeittresor** [comment]
i think i have a little workaround done for that 😉 https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/1158#discussion-7057249
